examples/passing/PolyLabels.purs

```diff
@@ -0,0 +1,65 @@
+module Main where
+
+import Prelude
+import Control.Monad.Eff
+import Control.Monad.Eff.Console
+import Data.Symbol (class IsSymbol, SProxy(..), reflectSymbol)
+
+foreign import unsafeGet
+  :: forall r a
+   . String
+  -> Record r
+  -> a
+
+foreign import unsafeSet
+  :: forall r1 r2 a
+   . String
+  -> a
+  -> Record r1
+  -> Record r2
+
+get
+  :: forall r l a
+   . IsSymbol l
+  => Lookup l r a
+  => SProxy l
+  -> Record r
+  -> a
+get l = unsafeGet (reflectSymbol l)
+
+set
```

Comment:
Yeah that's quite nice, and it also covers `Singleton` with: `Singleton l t r = Lookup l t r ()`.
Although the fun deps are slightly different; would it also make sense for `Lookup` to have `r1 r2 -> l a`?